app/controllers/internal/configs_controller.rb

```diff
@@ -11,7 +11,9 @@ def create
     clean_up_params
 
     config_params.each do |key, value|
-      if value.respond_to?(:to_h)
+      if value.is_a?(Array)
```

Comment:
arrays also respond to `to_h` as they could be arrays of array to build a hash, this is not a case, so we need a special condition for them